k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/fluent-bit/fluent-bit.yaml

```diff
@@ -181,7 +181,7 @@ data:
         Name                tail
         Tag                 host.dmesg
         Path                /var/log/dmesg
-        Parser              syslog
+        Parser              dmesg
```

Comment:
Please make changes to remove parser and switch to telling fluent bit to just put the whole log in a message key: https://github.com/aws-samples/amazon-cloudwatch-container-insights/pull/108#issuecomment-1353804758

Reply:
@PettitWesley done.

Now I'm using the "Key message" and I removed the parser previously added.
I've not tested it, I don't have a 2 minutes way to test it